README.md

```diff
@@ -1,5 +1,15 @@
 # GOV.UK coronavirus - find support
 
+⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️
+
+<p style="text-align: center">This repository is the old Coronavirus Find Support service - https://find-coronavirus-support.service.gov.uk.</p>
+
+<p style="text-align: center; font-weight: bold;">It is no longer deployed or in active development.</p>
```

Comment:
Think you can make thing bold in markdown `**It is no longer deployed or in active development.**`

Reply:
Hi @theseanything - however, markdown doesn't support text align center.

Reply:
⬆️  Ignore this - looks like GitHub ignores the center anyway.  Fixed.